extensions/Khronos/KHR_draco_mesh_compression/README.md

```diff
@@ -0,0 +1,138 @@
+# KHR_draco_mesh_compression
+
```

Comment:
All Khronos specifications should contain the standard Copyright Header:
https://www.khronos.org/members/login/groups/Agreements%20and%20Licenses/Open%20Source%20Repository%20Resources/Khronos%20Specification%20Copyright%20License%20Header%20V3%20May17.txt

Reply:
As the Draco extension is referring to significant external documents – namely the Bitstream specification we should also include, and be consistent with the additional  header wording for normative clarity:
https://www.khronos.org/members/login/groups/Agreements%20and%20Licenses/Open%20Source%20Repository%20Resources/Khronos%20Copyright%20License%20Header%20Addition%20for%20NORMATIVE%20CLARITY%20Oct17.txt   Note - this has some boiler plate that needs to be tweaked to fit the spec.
 
The key text in the header is:
 
Some parts of this Specification are purely informative and do not define requirements
necessary for compliance and so are outside the Scope of this Specification.  These
parts of the Specification are marked by the "`Note`" icon or designated "`Informative`"
<replace/insert specific conventions for the specification here>.
 
And
 
Where this Specification includes normative references to external documents, only the
specifically identified sections and functionality of those external documents are in
Scope. Requirements defined by external documents not created by Khronos may contain
contributions from non-members of Khronos not covered by the Khronos Intellectual
Property Rights Policy.

Reply:
Done